packages/nvidia_nat_atif/src/nat/atof/events.py

```
@@ -0,0 +1,131 @@
+# SPDX-FileCopyrightText: Copyright (c) 2026, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""ATOF event models for the 7 NeMo-Flow event types.
+
+Mirrors the Rust event structs in ``crates/core/src/types/event.rs`` as
+standalone Pydantic models. The ``Event`` type is a discriminated union
+keyed on the ``kind`` field.
+
+See ATOF spec Sections 2–3.
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)